pyaedt/emit.py

```diff
@@ -45,44 +271,50 @@ class Emit(FieldAnalysisEmit, object):
         parameter works only in 2022 R2 and later. The remote server must be
         up and running with the command `"ansysedt.exe -grpcsrv portnum"`.
         If the machine is `"localhost"`, the server starts if it is not present.
-    port : int, optional
-        Port number on which to start the oDesktop communication on an already existing server.
-        This parameter is ignored when creating a new server. It works only in 2022 R2 or
-        later. The remote server must be up and running with the command `"ansysedt.exe -grpcsrv portnum"`.
     aedt_process_id : int, optional
         Process ID for the instance of AEDT to point PyAEDT at. The default is
         ``None``. This parameter is only used when ``new_desktop_session = False``.
 
     Examples
     --------
-    Create an instance of Emit and connect to an existing Emit
-    design or create a new Emit design if one does not exist.
-
+    Create an instance of Emit. You can also choose to define parameters for Emit here.
     >>> from pyaedt import Emit
-    >>> app = Emit()
-
-    Create a instance of Emit and link to a project named
-    ``"projectname"``. If this project does not exist, create one with
-    this name.
-
-    >>> app = Emit(projectname)
+    >>> aedtapp = Emit()
 
-    Create an instance of Emit and link to a design named
-    ``"designname"`` in a project named ``"projectname"``.
+    Typically, it is desirable to specify a project name, design name, and other parameters.
+    >>> aedtapp = Emit(projectname, designame)
 
-    >>> app = Emit(projectname,designame)
+    Once an instance of Emit is initialized, the schematic can be edited:
+    >>> rad1 = aedtapp.modeler.components.create_component("Bluetooth")
+    >>> ant1 = aedtapp.modeler.components.create_component("Antenna")
+    >>> if rad1 and ant1:
+    >>>     ant1.move_and_connect_to(rad1)
 
-    Create an instance of Emit and open the specified project,
-    which is named ``Myfile.aedt``.
-
-    >>> app = Emit("myfile.aedt")
+    Once the schematic is generated, the ``Emit`` object can be analyzed to generate
+    a revision. Each revision is added as an element of the ``Emit`` object member's
+    ``revisions_list``.
+    >>> aedtapp.analyze()
 
+    A revision within PyAEDT is analogous to a revision in AEDT. An interaction domain must
+    be defined and then used as the input to the run command used on that revision.
+    >>> domain = Emit.interaction_domain()
+    >>> domain.rx_radio_name = "UE - HandHeld"
+    >>> interaction = aedtapp.revisions_list[0].run(domain)
 
+    The output of the run command is an interaction type object. An interaction summarizes the interaction data
+    of whatever is defined in the interaction domain.
+    >>> instance = interaction.worst_instance(Emit.result_type().sensitivity)
+    >>> val = instance.value(Emit.result_type().sensitivity)
+    >>> print("Worst-case sensitivity for Rx '{}' is {}dB".format(domain.rx_radio_name, val))
```

Comment:
```suggestion
    >>> print("Worst-case sensitivity for Rx '{}' is {}dB".format(domain.rx_radio_name, val).)
```
Ending punctuation is needed in messages to make a complete sentence.